lib/puppet_litmus/rake_tasks.rb

```diff
@@ -94,6 +96,13 @@
           check_connectivity?(inventory_hash_from_inventory_file, target)
         end
       end
+    rescue PuppetLitmus::RakeHelper::LitmusTimeoutError
+      current_retry_count += 1
+      Rake::Task['litmus:tear_down'].invoke(target_names.first)
+      raise if current_retry_count >= retry_count
```

Comment:
should this be the below to allow for 3 retries?
```suggestion
      raise if current_retry_count > retry_count
```